SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -124,6 +249,13 @@ public async Task BeforeScenario()
             options.AddArguments("--window-size=1920,1080");
             options.AddArguments("--start-maximized");
             options.AddArguments("--headless");
+            //options.A "same-site-by-default-cookies", "2");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)